monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -240,6 +254,23 @@ enable_pod_token_automount $MON_NS deployment v4m-operator
 log_info "Deploying ServiceMonitors and Prometheus rules"
 log_verbose "Deploying cluster ServiceMonitors"
 
+if [ "$TRACING_ENABLE" == "true" ]; then
+  TEMPO_CHART_VERSION=${TEMPO_CHART_VERSION:-1.5.0}
+  log_info "Tracing enabled..."
+  # # Add the grafana helm chart repo
+  helmRepoAdd grafana https://grafana.github.io/helm-charts
+  helm repo update
```

Comment:
I think Curtis and I determined that the helm repo update commands are unnecessary after the calls to helmRepoAdd since the function either adds the repo for the first time or does a helm repo update on its own.  We can confirm that ande remove thiswhen we add support for air-gap.